src/main/java/net/glowstone/generator/structures/GlowTemple.java

```diff
@@ -26,8 +26,6 @@ public GlowTemple(World world, int chunkX, int chunkZ) {
     public GlowTemple(World world, Random random, int chunkX, int chunkZ) {
         this(world, chunkX, chunkZ);
 
-        types.put(Biome.OCEAN, TempleType.values()[random.nextInt(3)]); // fix for lack of biomes
```

Comment:
Already done is class static. At the time I made this, there was only the ocean biome on glowtone, it explains the line artifact, but the types list is already fed ;)